migrations/db/migrations/20250417190610_update_pgbouncer_get_auth.sql

```diff
@@ -0,0 +1,24 @@
+-- migrate:up
+
+create or replace function pgbouncer.get_auth(p_usename text) returns table (username text, password text)
+    language plpgsql security definer
+    as $$
+begin
+    raise debug 'PgBouncer auth request: %', p_usename;
+
+    return query
+    select 
+        rolname::text, 
+        case when rolvaliduntil < now() 
+            then null 
+            else rolpassword::text 
+        end 
+    from pg_authid 
+    where rolname=$1 and rolcanlogin;
+end;
+$$;
+
+-- from migrations/db/migrations/20250312095419_pgbouncer_ownership.sql
+grant execute on function pgbouncer.get_auth(p_usename text) to postgres;
```

Comment:
@kangmingtay We just need one more test to prove this is working correctly :pray: . Just copy/paste this query from vault and change it to use the `pgbouncer` schema:

https://github.com/supabase/postgres/blob/61ff753243078fe2b14e5b8f184f4a00c80342ee/nix/tests/expected/vault.out#L1-L27

This should show all the privileges on the pgbouncer schema objects.